tests/tests_pytorch/checkpointing/test_model_checkpoint.py

```diff
@@ -870,17 +870,19 @@ def validation_step(self, batch, batch_idx):
         "enable_progress_bar": False,
         "enable_model_summary": False,
     }
-    trainer = Trainer(**trainer_kwargs, callbacks=[checkpoint_callback])
+    trainer = Trainer(**trainer_kwargs, default_root_dir=tmpdir, callbacks=[checkpoint_callback])
```

Comment:
without this change tensorboard would initialize its version by incrementing last version number from other tests (should be init as `version_0` for assert below)